schema/defs.json

```diff
@@ -41,10 +41,10 @@
             "minimum": 0,
             "maximum": 18446744073709552000
         },
-        "percent": {
+        "maximum": {
             "type": "integer",
             "minimum": 0,
```

Comment:
Another possibility, now that this shares the same 1–10k range as `cpuShares`, would be to rename the `cpuShares` entry in `defs-windows.json` to [`basisPoint`][1] or some such (`permyriad`?  `perTenThousand`?).  That's generic enough that it could live in `defs.json`

[1]: https://en.wikipedia.org/wiki/Basis_point